\documentclass[12pt]{article}
 \usepackage[hcentering,bindingoffset=20mm]{geometry}
 \usepackage{placeins}
 \usepackage[numbib]{tocbibind}
 \usepackage{rotating}
\usepackage[square,sort,comma,numbers]{natbib}
 \usepackage{graphicx}
 \usepackage{tabularx}
% \usepackage{datetime2}
 \linespread{1.3}
 \usepackage{gensymb}
\usepackage{longtable}
 \usepackage{lscape}
 \usepackage{url}
 \addtolength{\textwidth}{2cm}
 \addtolength{\hoffset}{-1cm}
 
 
 \addtolength{\textheight}{2cm}
 \addtolength{\voffset}{-1cm}
 \setlength{\parindent}{0pt}
 

\title{Trial by phylogenetics - how model and data selection impacts the elucidation of the evolution of the Gonyaulacales (Dinophyceae). $^{1}$}
\author{Key words: Bayesian inference, model selection, Gonyaulacales,\\ phylogenetics, transcriptomics}

\date{}

\begin{document}
\maketitle

%Timestamp \DTMnow UTC %sorry for some reason datetime2 package is breaking the rendering process - the tracklang dependency got updated or something and it's throwing a hissy fit. After about an hour of trying to fix it, I think I should be looking at the manuscript instead =\

\paragraph{}Anna Liza Kretzschmar$^{2}$\\
The ithree institute, University of Technology Sydney, Ultimo, 2007 NSW, Australia, anna.kretzschmar@uts.edu.au
\paragraph{}Mathieu Fourment \\
The ithree institute, University of Technology Sydney, Ultimo, 2007 NSW, Australia
\paragraph{}Shauna Murray\\ 
Climate Change Cluster (C3), University of Technology Sydney, Ultimo, 2007 NSW, Australia
\paragraph{}Arjun Verma\\
Climate Change Cluster (C3), University of Technology Sydney, Ultimo, 2007 NSW, Australia
\paragraph{}Aaron E. Darling \\
The ithree institute, University of Technology Sydney, Ultimo, 2007 NSW, Australia
\newpage
\section{Abstract}
\newpage

%General comments AD:
%new: line wis=dth: 8, text size: 30
%  bit of text in methods I think on compute speed but I didn't see that get revisited in results or discussion (unless I missed it)

\section{Introduction}
Historically, availability of genetic data to infer evolutionary relationships has been a limiting factor for phylogenetics.
Now, the breadth of publicly available data sets generated by high throughput DNA sequencing techniques allow for an increasingly detailed investigation into the evolutionary relationships between organisms.
The quest to untangle the evolutionary relationships among organisms is an ongoing process that can help inform a broad range of fields, for example epidemiology, toxicology and ecological interactions \cite{mctavish2017and}.
A limiting factor for our analysis of evolutionary relationships is, commonly, the computational methods and available infrastructure used to analyze the data sets. 
%MF operator? usually we talk about practitioners or users
Another is that the methods and models applied to the data set can influence the resulting phylogeny and there needs to be a solid understanding of the methods, including their shortcomings, by the operator.
An example of the breadth of publicly available data is the Marine Microbial Eukaryote Transcriptome Sequencing Project (MMETSP), which sequenced transcriptomes of over 650 marine eukaryotic microbes \cite{keeling2014marine}. 
This project focused on a group of understudied organisms which are abundant and play vital roles in the marine environment, from geochemical cycling, predation to symbiosis \cite{gomez2005list,gomez2012quantitative}. 
This data set offers an excellent opportunity to explore the evolutionary relationships between these taxa through phylogenetics. 

Central to phylogenetic inference is the relationship of two characters from a common ancestor, which is called homology \cite{fitch2000homology}. 
There are several types of homology, related to how the characters diverged, and determining through which mechanisms comparative traits/genes evolved is essential for choosing the correct inference model. 
Orthologs is the case where the divergence of two gene copies followed a speciation event \cite{fitch1970distinguishing}. 
Paralogs are two gene copies whose divergence is preceded by gene duplication \cite{fitch1970distinguishing}.
Xenologs are gene pairs which, additional to divergence, have undergone transfer through horizontal gene transfer \cite{darby2016xenolog}.
The distinction between these cases is essential in identifying gene candidates that are informative for species evolution inference, as the selection of orthologs ensures the inclusion of a signal that is based on the speciation of the taxa examined, while selection of paralogs confounds that signal by including information that does not pertain to the speciation of the taxa. 
As gene duplication and subsequent loss commonly occur over the course of evolution and speciation, the identification of genes that have orthologous relationships is more difficult than may seem apparent from the definition \cite{gabaldon2008large}. 

Once orthologs have been identified there are further issues that can arise and impact the veracity of the phylogenetic inference.
Two common, well characterized types of errors are random (sampling error) and systematic errors. 
The former arises from the data, as individual gene histories may differ to the species tree. 
With a small number of genes, this error can reduce the confidence (through node support values) of the topology, and in extreme cases skew the inference away from determining a good approximation of the species tree entirely. 
%MF a reference would be good here:
Increasing the number of genes directly reduces the impact this error has on the analysis. 

% AD 181018: The following reads well
Conversely, systematic errors arise due to the misspecification of the model used for the inference where an incorrect species tree topology results. 
In this case, an increase in data set size can exacerbate systematic error rather than reduce it as would happen with random errors (see Box 1). 
In the presence of this type of systematic error, the resulting inference can be positively misleading,
 with high clade support values for the incorrect tree topology, obfuscating the presence of the error \cite{jeffroy2006phylogenomics,roch2015likelihood,kubatko2007inconsistency}. 

In summary, common problems in carrying out a species tree inference arise from:\\
\begin{enumerate}
\item Selection of paralogs. 
If genes with different evolutionary histories are selected, the gene tree will not reflect the history of either paralog and be nonsensical for species tree inference;
\item Concatenation of genes. 
Can be statistically inconsistent estimator of the species tree due to incomplete lineage sorting (ILS) and concatenation acting as imperfect estimator of species tree topology \cite{roch2015likelihood};
\item Inference of model adequacy from bootstrap values. 
%MF is it also a problem using Bayesian inference?
Kubatko et al. (2007) demonstrated high bootstrap support under maximum likelihood (ML) inference for incorrect species trees with concatenated gene sets as input \cite{kubatko2007inconsistency}. 
As high bootstrap values are often used as an indicator for robust species topology resolution, this fallacy is particularly problematic if the reader/operator is unfamiliar with the statistical phenomenon.
\end{enumerate}

In this study we explore the application of data analysis techniques which attempt to mitigate several of the pitfalls in species tree inference, beyond what has previously been applied in the study of protist phylogeny.
The sequence data is prepared using a workflow that selects single copy genes. 
Next, we evaluate the impact of model and data selection on the resulting phylogenetic inference. 
%MF I would move this sentence to the Material and Methods:
Source code for the scripts to assemble RNA-seq data sets, to identify and extract single copy genes across input taxa with extensive paralogy, and to align selected genes ready for Bayesian inference (BI) phylogenetics is provided under an open source license. 
Finally, we applied the methodology to a group of organisms notorious for their extensive paralogy - the Gonyalacales (phylum: Dinoflagellata) (see box 2 for further information on the gonyaulacales).
%MF I would finish the intro with a sentence about what you have discovered (results)

%can I include this again now ^___^
%Within this framework, the dinoflagellates and in particular the gonyaulacales hold a special place in hell. 
%This author postulates that there is a Dante-esque lower circle especially reserved for protist microbiologists and taxonomists whose willfully laisez-faire approach abuses mathematical models to spew forth statistically inconsistent abominations of their complex evolutionary history.
% AD 181018: wow, that's beautiful.

%TODO limited taxon sampling section, not sure whether to include this here or in the discussion is sufficient?
%Heath 08: Because  parametric  methods,  such  as  maximum  likelihood,  incorporate  models  that  account  for  unobserved  substitutions,  these  methods  are less prone to the effects of long-branch attraction, as  long  as  the  models  of  evolution  are  adequate.  However, enough taxa must be sampled to parameter-ize  these  models  effectively  (Pollock  et  al.,  2002). In  addition,   longer   branches   require   more   accurate   models   of   evolution   (because   more   unobserved   changes  must  be  inferred),  so  increased  taxon  sam-pling (which breaks up long branches) greatly benefits parametric  methods  as  well  as  nonparametric  meth-ods. 
% Xia 06 Topological Bias in Distance-Based Phylogenetic Methods: Problems with Over- and Underestimated Genetic Distances
% Pollock Increased Taxon Sampling Is Advantageous for Phylogenetic Inference
\subsubsection*{Box 1: Statistical nomenclature \& errors this study seeks to address}
For in depth explanations see \cite{yang2014molecular}.
\begin{itemize}
  \setlength{\itemsep}{1pt}
  \setlength{\parskip}{0pt}
  \setlength{\parsep}{0pt}
\item \textbf{Heterotachy:} change in evolutionary rates of sites over time, specific to lineage(s).
\item \textbf{Potential statistical error types:}
\begin{enumerate}
  \setlength{\itemsep}{0pt}
  \setlength{\parskip}{0pt}
  \setlength{\parsep}{0pt}
\item random. Sampling based error which decreases and approaches zero as the size of the data set approaches infinity.\\
\item systematic. Arises from incorrect model assumptions or problems with the model itself. 
Error type persists and increases as data set size approaches infinity. 
If strong, can override true phylogenetic signal.
\end{enumerate}
\item \textbf{Incomplete lineage sorting (ILS):} discordance of gene evolutionary history with the species evolutionary history causing the phylogenetic species tree to be misinferred. 
Difference in the topology of a gene tree compared to the species evolution can arise from the coalescence of those orthologs prior to the species divergence, where in effect the ancestral populations contain two or more already-diverged copies of the gene across one or more species divergence points. 
Another mechanism is the introduction of a copy of the gene which is not based on ancestral inheritance, such as horizontal gene transfer or hybridization.
\item \textbf{Long branch attraction (LBA):} placement of two heavily divergent but distantly related sequences with each other. 
The model is unable to extract evolutionary signal due to the number of mutations that have occurred, so places the two taxa together. 
Also called the Felsenstein zone.
\end{itemize}


\subsubsection*{Box 2: Who/what are the Gonyaulacales?}
%TODO Aaron and Mathieu, as non protistologists is this section adequate in explaining why the gonyaulacales were chosen for this work, or would it help to include some of the commented out text? Would the commented out sections be helpful?
The gonyaulacales are an order within the super-phylum Alveolata and sub-phylum Dinoflagellata, which are an ancient eukaryotic lineage \cite{moldowan1998biogeochemical}. 
They play a role in several important ecological processes in aquatic environments where they cover a diverse array of niches such as symbionts, parasites and autotrophs. 
%MF why using "&" instead of "and", it seems informal
Some taxa can cause harmful algal blooms through proliferation (by restricting light \& nutrient availability to other organisms) and/or neurotoxin production (e.g. causing paralytic shellfish poisoning, ciguatera fish poisoning) \cite{murray2016unravelling}.
Dinoflagellates possess large genomes (estimated size range 3.7 to 220 Gbp), with extensive paralogy and repetitive short sequences  \cite{casabianca2017genome,murray2016unravelling}. 
In particular the paralogy has proven problematic for efforts investigating the genetic content and structure of the dinoflagellates, as this feature has prevented the assembly of genomes apart from two draft genomes for \textit{Symbiodinium} which posses some of the smaller genomes \cite{shoguchi2013draft,lin2015symbiodinium}. 
For a review on the genetic features of dinoflagellates see \cite{murray2016unravelling}. 
While the evolutionary relationship of most orders within the dinoflagellates has been inferred with consistently high support values, one order has often escaped elucidation - the gonyaulacales. 
As neurotoxin production is prevalent in this order, the evolution of the order is of interest to provide a frame of reference for future investigations into how the toxins have evolved \cite{shalchian2006combined,zhang2007three,saldarriaga2004molecular,hoppenrath2010dinoflagellate,murray2005improving}. 
%Specifically the ciguatoxins produced by some \emph{Gambierdiscus} spp. are of interest as the causative agent of ciguatera fish poisoning, a neglected tropical disease with global implications \cite{globalcig}.%, which is predicted to increase in prevalence as climate change progresses \cite{}.

\newpage
\section{Materials and methods}
\subsection*{Culture conditions}
\FloatBarrier
Cultures were isolated from locations as per Table S1 and clonal cultures established by micropipetting single cells through sterile seawater. 
Clonal cultures were maintained in F/10 medium and maintained at temperatures indicated in Table S1. 

\subsection*{RNA isolation}
\emph{Gambierdiscus} spp. and \emph{Thecadinium} cf. \emph{kofoidii} were harvested during late exponential growth phase by filtration onto 5 $\mu$m SMWP Millipore membrane filters (Merck, DE) and washed off with sterile seawater. 
Cells were pelleted via centrifugation for 10 minutes at 350 rcf. 
The supernatant was decanted and 2ml of TRI Reagent (Sigma-Aldrich, subsidiary of Merck, DE) was added to the pellet and vortexed till dissolved. 
Samples were split in two and transferred to 1.5ml eppendorf tubes. 
Cellular thecae were ruptured by three rounds of freeze-thaw, with tubes transferred between liquid Nitrogen and 95 $^{\circ}$C. 
RNA was extracted as per the protocol for TRI Reagent. 
RNA eluate was purified with the RNeasy RNA clean up kit EXACT NAME as per protocol (Qiagen). 
%TODO above
DNA was digested with TurboDNAse (Life Technologies, subsidiary of Thermo Fischer scientific, AU). 
RNA was quantified with a Nanodrop 2000 (Thermo Scientific, Australia) and frozen at -80 $^{\circ}$C until sequencing.
%TODO insert centrifuge, kit name deets
 
\subsection*{Library preparation and Sequencing}
The quality of samples was assessed via an Agilent 2100 Bioanalyzer. 
Paired-end sequencing was performed with a NextSeq 500 High Output run at the Ramaciotti Center (UNSW, AU) with 75bp read length for \emph{G. lapillus} and \emph{G.} cf. \emph{silvae}; and 150bp read length for \emph{G. carpenterii}, \emph{G. polynesiensis} CG15 and \emph{T.} cf. \emph{kofoidii}.

\subsubsection*{Publicly available transcriptome libraries}
The \emph{Gambierdiscus excentricus} VGO790 transcriptome was downloaded from NCBI under accession ID SRR3348983 \cite{kohli2017role}. 
\textit{Coolia malayensis}, \textit{Ostreopsis ovata}, \textit{Ostreopsis rhodesae} and \textit{Ostreopsis siamensis} transcriptomes were supplied by Arjun Verma (Verma 2018, in prep). 
Accession numbers available in table \ref{tbl:asmstats}. 
RNA-seq libraries for all remaining transcriptomes were generated by, and downloaded from, the Marine Microbial Eukaryote Transcriptome Sequencing Project \citep{keeling2014marine}.

\subsection*{Transcriptome processing scripts}
%MF It is probably better to give the full github address \url{https://github.com/hydrahamster/gonya_phylo}
The analysis work flow was written in the Nextflow language \cite{nextflow} and is available on Github under hydrahamster/gonya\_phylo. 
The workflow is separated into two parts, and packages within the scripts are written in bash, Python 2.7 \cite{python} and pandas \cite{pandas}, and can be found on Github. 
Our data sets were analyzed using a Genomics Virtual Lab (GVL) \cite{afgan2015genomics} instance in the NeCTAR cloud.
\subsubsection*{Transcriptome assembly}
Individual RNA sequencing libraries are the input, which are then processed through FastQC \cite{fastqc} for quality metrics, sequences are trimmed with Trimmomatic (\texttt{LEADING:3 TRAILING:3 SLIDINGWINDOW:4:5 MINLEN:25}) \cite{bolger2014trimmomatic} and assembled with Trinity v2.4.0 (default settings for paired end libraries) \cite{haas2013novo}. 
Assemblies were then processed with BUSCOv2 with the protist specific library \cite{simao2015busco}.
The RNA libraries with 150bp reads generated as part of this study were also subjected to Digital Normalization \cite{diginorm} prior to assembly, to pool identical transcripts before assembly which were then used for downstream analysis.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
\subsubsection*{Construction of multiple alignments}
The BUSCOv2 output from all transcriptomes from the previous step forms the input for identification of single copy genes and construction of multiple alignments. 
Any genes that were present as single copies in at least 75\% of the transcriptomes were indexed, the corresponding contig extracted from the assemblies, aligned with hmmer3.1b2 \cite{eddy2015hmmer} and unaligned regions trimmed.
If several candidate sequences are processed for the same organism, a warning message in the terminal window alerts the user before proceeding. 
The output for this section was used as a basis for single copy gene phylogenetic inferences in subsequent sections.

\subsection*{Assembly analysis}
Contigs from assemblies were clustered with cd-hit with the flags \texttt{-T 10 -M 5000 -G 0 -c 1.00 -aS 1.00 -aL 0.005} \cite{fu2012cd}. 
Protein coding regions within the clusters were predicted with Transdecoder \cite{haas2016transdecoder}. 
Amino acid clusters were clustered again with cd-hit with the flags as previously except \texttt{-c 0.98}.
Protein sequences were analyzed with interproscan v5.27 with local lookup server \cite{quevillon2005interproscan}.

\subsection*{Phylogenetic inferences}
\subsubsection*{Ribosomal DNA based inference}
Ribosomal DNA (rDNA) sequences for the small subunit (SSU) region as well as the D1-D3 large subunit (LSU) region were acquired from NCBI \cite{coordinators2017database} or the SILVA rRNA database project \cite{silvaproj}, accession IDs in Table S3. 
Individual genes were aligned using the MUSCLE algorithm for a maximum of 8 iterations \cite{edgar2004muscle}, then were concatenated in Geneious v11.3 \cite{kearse2012geneious}.
ML inference was obtained using RaxML \cite{stamatakis2014raxml} with the GTR and GAMMA flags, with 100 bootstrap replicates.

\subsubsection*{Concatenated single copy gene based inference}
%MF move HPC and GPU to the end
Single copy gene alignments were concatenated and both ML \& BI analyses were executed on the University of Technology Sydney's High-performance computing cluster (HPCC). 
GPU processing units were either Nvidia Tesla K80 or a Tesla P100. 
Amino acid substitution model selection was carried out with ProtTest3 with the Bayesian Information Criterion as well as the log likelihood \cite{darriba2011prottest,guindon2003simple}. 
The best-fit model for the data set identified by both criteria was VT followed by LG, however neither are available in BEAST2 so the third best model, WAG, was chosen for analysis. 
\paragraph*{Maximum likelihood with concatenated sequences.}
ML inference was run as described in the previous section, with the PROT, GAMMA and WAG flags.
\paragraph*{Bayesian inference with concatenated sequences.}
BI was run in BEAST2 with the Gamma site model with 4 discrete categories under the WAG substitution model \cite{whelan2001general}. A local random clock was used under the birth-death model %TODO insert nuber of chains etc
 

\paragraph*{Bayesian probability under the multispecies coalescent.}
Bayesian inference of the species tree was carried out under the *BEAST2 model in BEAST2 \cite{bouckaert2014beast}. 
The analysis was performed with the WAG amino acid substitution model \cite{whelan2001general} and with a Gamma distribution with four rate categories. 
A random local clock was employed \cite{drummond2010bayesian}. 
%MF move HPC to the end
Computation was carried out on the HPCC and was accelerated using BEAGLE \cite{ayres2011beagle} on the GPU. 
Posterior distributions of parameters were approximated after 300,000,000 generations of MCMC, subsampled every 5,000 generations with a burn-in of 15\%. 
The inference was run four times to evaluate convergence of parameters, then log and tree files were merged. 

%\subsection*{stepping stone analysis}
%TODO put details in in the distant future

%LK: I've just taken this out because I don;t really have a proper comparison. The CPU run took so long I had to kill it.
%\subsection*{Inference run time comparison}
%A subset of 14 taxa was selected for a comparison between CPU and GPU inference run time comparison. 
%The transcriptome libraries were fed through the single copy gene extraction script as described, which was set to extract single copy genes present in all of the taxa. 
%The resulting alignments were run on *BEAST either on the CPU or GPU.

\subsubsection*{Generation of figures}
Tanglegrams were generated with Dendroscope v3.5.9 \cite{huson2007dendroscope}; images were edited in GIMP \cite{gimp}.

%MF I would add section here about implementation. You can talk about your scripts, github, and the hardware you used (i.e. HPC, GPUs...). 
\newpage
\section{Results}
\subsection*{Transcriptomes overview}
RNA-seq libraries generated in this study are available on NCBI's sequence read archive under the project ID SRP134273.
Sequencing of transcriptomes for \emph{Gambierdiscus} spp. and \emph{T.} cf. \emph{kofoidii} generated data sets ranging in size from 143,155,667 to 233,822,334 reads, resulting in 97,634 to 191,224 assembled contigs (table ~\ref{tbl:asmstats}). 
Clusters with gene ontology (GO) annotations made up 30.9\% to 34.8\% of the total clusters. 

\FloatBarrier
\begin{longtable}{  | p{3cm} |p{2.2cm} | p{2.2cm} | p{2.2cm} | p{2.2cm} | p{2.2cm} |}
\caption{Summary of transcriptome sequencing and assembly statistics.}\\
\hline
\label{tbl:asmstats}
\emph{Sequences:}&\emph{G. carpenteri}&\emph{G. lapillus}&\emph{G. polynesiensis}&\emph{G.} cf. \emph{silvae}&\emph{T.} cf. \emph{kofoidii}\\
\hline
 \multicolumn{6}{| c |}{Sequencing}\\
 \hline
\textbf{SRA accession}&SRR6821720&SRR6821722&SRR6821723&SRR6821721&SRR6821724\\
\hline
\textbf{Raw sequencing reads}&186,422,744&145,366,966&217,031,342&143,155,667&233,822,334\\
\hline
%\textbf{Size (Gb)}&16.92&&19.51&&21.08\\
%\hline
 \multicolumn{6}{| c |}{Assembly}\\
 \hline
 \textbf{Contigs \#}&105,464&148,972&114,622&191,224&97,634\\
\hline
\textbf{Average length (bp)}&607&1,139&633&953&581\\
%\hline
%\textbf{Minimum length (bp)}&201&201&201&201&201\\
\hline
\textbf{Maximum length (bp)}&7,448&12,370&6,608&8,198&7,922\\
\hline
  \multicolumn{6}{| c |}{Transcript clustering \& annotation}\\
\hline
\textbf{\# clusters}&139,699&92,418&139,487&107,766&116,468\\
\hline
\textbf{Contigs with GO annotations}&44,167&32,140&43,098&34,201&37,656\\ %cut -f 3 my.gff | sort | uniq | wc -l .. maybe diff to '3' but hey
\hline
%\textbf{with full annotations}&&&&&\\
%\hline
%\textbf{with Enzyme Codes}&&&&&\\
%\hline
%\textbf{mapped to KEGG pathways}&&&&&\\
%\hline
\end{longtable}

\subsection*{BUSCO output}
Assemblies were searched with BUSCOv2 for 234 candidate single copy genes and homologs to these single copy genes were extracted. 
The single copy genes acquired though the BUSCO HMMER libraries curated for protists are reported in Table S2, as well as accession numbers and identifiers for each transcriptome. 
Single copy genes for each transcriptome used in this study are available on Zenodo DOI:
%TODO single copy genes need ot go up on Zenodo, then link DOI here

\subsection*{Phylogenetic inferences}
%MF Bayesian support values tend to overestimate support so I am not sure if it makes sense to have this straightforward relationship between posterior prob and bootstrap values.. Although I don't know if there is a better way
%MF if you talk about bootstrap proportions, the support values would be between 0 and 1, like the posterior probabilities. It would simplify the sentence below:
Support for branches was interpreted as follows, for ML and BI, respectively: 100\%/1.0 was considered fully supported, above 90\%/0.9 was very well supported, 80\%/0.8 and above was interpreted as relatively well supported and above 50\%/0.5 was considered weakly supported but below was considered unsupported.
As \emph{Azadinium spinosum}, \emph{Dinophysis acuminata} and \emph{Karenia brevis} are members of different orders (Dinophyceae ordo incertae sedis, Dinophysiales \& Gymnodiniales respectively) and consistently resolve outside of the gonyaulacales in phylogenetic analyses, their placement as an outgroup was considered a given for this study. 
Therefore, the branch separating these taxa from others was used to root the tree in subsequent analyses where rooting was required.
\subsubsection*{rDNA based phylogeny}
\FloatBarrier 
All nodes were supported, with a range of certainty.
The following refers to Fig. ~\ref{fig:rdna}.
Species within the genera \emph{Gambierdiscus} and \emph{Ostreopsis} resolved with their sister species with full support. 
Within the \emph{Gambierdiscus} clade, nodes are either weakly supported or fully supported. 
The two species of \emph{Alexandrium} resolve as well supported closest relatives, but do not form an individual clade. 
Deeper nodes were supported but with less certainty than the nodes near the tips. 
Two distinct clades can be observed from the topology: One including \emph{Alexandrium}, \emph{Coolia} and \emph{Ostreopsis}; another with only \emph{Gambierdiscus}. 
Sister to these clades, in descending order, was \emph{Pyrodinium}, \emph{Ceratium} and \emph{Gonaulax}, \emph{Protoceratium} and \emph{Thecadinium}. 
The outgroup was relatively well supported and included \emph{Crypthecodinium}. 
Support for deeper nodes varied from weak to well supported.

\begin{figure} 
\includegraphics[scale=.4]{figures/rDNA-ML.png} 
%MF  "n=16" is easier to understand than "#16"
%MF The scale represents the *expected* number of  amino acid  substitutions per site
%MF in the figure you talk about incertae sedis but in the text you mention Crypthecodinium. It is probably a taxonomy thing I am not aware of but are they the same things?
\caption{Maximum likelihood phylogenetic inference of ribosomal DNA genes. Concatenation of small subunit rDNA and D1-D3 region large subunit rDNA. Accession numbers for concatenated genes in Table S3. Gonyaulacales (\#16) in purple, outgroups (\#3) in light blue and taxa \textit{incertae sedis} (\#1) in teal. Topology was rerooted on branch separating outgroup taxa with the gonyaulacales. The scale represents number of substitutions per site.} 
\label{fig:rdna}
\end{figure} 
\FloatBarrier

\subsubsection*{Concatenated single copy gene based phylogeny inferred with ML}
\FloatBarrier
All nodes except one within the \emph{Gambierdiscus} species cluster resolved as relatively well supported. 
Topological description in this section refers to Fig. ~\ref{fig:SCconcatML}. 
Species of the genera \emph{Alexandrium}, \emph{Gambierdiscus} and \emph{Ostreopsis} cluster as individual clades with their sister species.  
The topology shows three distinct, well supported clades: 
One encompassing \emph{Alexandrium}, \emph{Coolia} and \emph{Ostreopsis}; another which only contains \emph{Gambierdiscus}; and one which includes \emph{Pyrodinium}, \emph{Gonyaulax} and \emph{Protoceratium}. 
Sister to these clades is \emph{Thecadinium}, followed by \emph{Ceratium}.
The split of the outgroup was fully supported, while the internal nodes resolved very well supported. 
\emph{Crypthecodinium} resolved within the outgroup, sister to \emph{Karenia}. 
Other deeper nodes were well supported.
 
\begin{figure} 
\includegraphics[scale=.45]{figures/SC-concat-ML_WAG.png} 
\caption{Maximum likelihood phylogenetic inference of concatenated single copy gene set (62 single copy genes from 20 taxa). Gonyaulacales (\#16) in purple, outgroups (\#3) in light blue and taxa \textit{incertae sedis} (\#1) in teal. Topology was rerooted on branch separating outgroup taxa with the gonyaulacales. The scale represents number of substitutions per site.} 
\label{fig:SCconcatML}
\end{figure} 
\FloatBarrier

\subsubsection*{Concatenated single copy gene based phylogeny inferred with BI}
\FloatBarrier 
%TODO Running with stepping stone ~~~ check out trees closer to posterior for prev ss sampling run in /shared/homes/s1/Liza/concat-bollocks/test // um... which on edid I use??
All nodes resolved with full support, except one node within the genus \textit{Gambierdiscus} which was very well supported. 
The following descriptions are based on Fig. ~\ref{fig:SCconcatBI}. 
%MF I am not sure I understand. Do you mean that each genus is monophyletic?
The species in the genera \textit{Alexandrium}, \textit{Gambierdiscus} and \textit{Ostreopsis} all resolved with full support within their respective genera clades. 
The overall topology of the gonyaulacales resolved as three clades with \textit{Thecadinium} and then \textit{Ceratium} as ancestral lineages. 
\textit{Alexandrium}, \textit{Coolia} and \textit{Ostreopsis} clustered together, followed by \textit{Gambierdiscus} on their own in a sister clade. 
The third clade encompasses \textit{Gonyaulax}, \textit{Protoceratium} and \textit{Pyrodinium}. 

\begin{figure} 
\includegraphics[scale=.34]{figures/SC-concat-BI.png} 
%MF rerooting a rooted tree is strange. Rerooting a tree from MrBayes without a clock would make more sense
\caption{Bayesian phylogenetic inference of concatenated single copy gene set (62 single copy genes from 20 taxa). Gonyaulacales (\#16) in purple, outgroups (\#3) in light blue and taxa \textit{incertae sedis} (\#1) in teal. Topology was rerooted on branch separating outgroup taxa with the gonyaulacales. The scale represents number of substitutions per site.} 
\label{fig:SCconcatBI}
\end{figure} 
\FloatBarrier

\subsubsection*{Single copy gene based phylogeny under MSC}
\FloatBarrier 
All nodes except one within the outgroup clustering resolved.
The following description is based on the topology in Fig. ~\ref{fig:SCmscBI}. 
Species of \emph{Alexandrium}, \emph{Ostreopsis} and \emph{Gambierdiscus} resolved well or fully supported within their genus clades. 
The topology within the gonyaulacales resolves as three clades: 
one fully supported encompassing \emph{Alexandrium}, \emph{Coolia} and \emph{Ostreopsis};
a well supported clade with \emph{Gambierdiscus} and \emph{Pyrodinium}; 
and a weakly supported clade including \emph{Ceratium}, \emph{Gonyaulax}, \emph{Protoceratium} and \emph{Thecadinium}. 
The outgroup clustered together with high support. 
\emph{Crypthecodinium} resolved as a sister taxon to the outgroup. 
Other deeper nodes were well or fully supported.

\begin{figure} 
\includegraphics[scale=.25]{figures/Aug2_20-taxa-combined-fig_MCC_trees.png} 
%MF this rerooted tree looks very different from the previous tree as it still looks ultrametric. Does it mean MSC got the rooting right, unlike the concatenated tree?
\caption{Bayesian phylogenetic inference of gonyaulacales species tree under the multipsecies coalescent with 62 single copy genes from 20 taxa. Gonyaulacales (\#16) in purple, outgroups (\#3) in light blue and taxa \textit{incertae sedis} (\#1) in teal. Topology was rerooted on branch separating outgroup taxa with the gonyaulacales. The scale represents number of substitutions per site.} 
\label{fig:SCmscBI}
\end{figure} 
\FloatBarrier

\newpage
\section{Discussion}
%explanation of model choice, history of analysis, shortcommings
Phylogenetic inference has become a key aspect of exploring the evolutionary relationships between organisms, with applications in pathology, ecology, investigating adaptive traits and many more \cite{heath2008taxon}. 
%MF this sentence looks strange:
An increase in sequencing throughput and reduction of associated cost has seen an increase in large scale sequencing initiatives such as MMETSP, which covers a relatively uncharacterized group of marine microbial eukaryotes \cite{keeling2014marine}.
However, the methodologies used for investigating the evolutionary relationships between these taxa remain an obstacle, as the choice of input data and method employed influences the outcome of the inference. 
To address this, a synopsis on RNA library pre-processing, assembly parameter selection, a method for single copy gene selection, and synthesis of phylogenetic inference models availability and selection is presented in this study - as well as possible shortcomings of the parameters and methods selected. \\
Dinoflagellates are notorious for their extensive genomes with suspected whole or partial genome duplication and potential cDNA retro-insertion into the genome \cite{van2009florida,beauchemin2012dinoflagellate,slamovits2008widespread,hou2009distinct,lin2011genomic}. 
This causes unusually large gene copy numbers and extensive paralogy. 
With this in mind, the gonyaulacales (an order within the dinoflagellates, see Box 1) presented as an ideal challenge to test the parameters of interest.

The phylogenetic inference for gonyaulacales that resulted from the workflow we developed, which implements several of the most recent innovations in analytical methodology, resolved within-genus relationships well and shows high P.P. throughout the species tree (Fig. ~\ref{fig:SCmscBI}). 
The inferred species tree topology follows a broad revised taxonomic classification of the gonyaulacales based on morphological characteristics \cite{hoppenrath2017dinoflagellate} and will be used as a point of comparison to results from other commonly employed methods in later sections. 
The scripts which form the basis of this study are publicly available through github and the single copy genes used to infer the species evolution in this study, as well as the XML input and log files for the *BEAST2 runs, are available on zenodo. 
The authors hope the process presented here is transparent and reproducible for those with basic programming skills. 
%other factors to consider.. qiality of transcriptome etc
\subsection*{Considerations for data set selection and pre-processing}
\subsubsection*{Quantity of taxa in phylogenetic inference}
Extensive coverage of taxa within the group of organisms examined is necessary to give useful insight into the evolutionary inter-taxon relationships.
Two concerning phenomena that can confound the veracity of conclusions drawn from phylogenetic inference are ILS and LBA. 
The risk of LBA artefacts can be reduced by denser taxon sampling to break up long branches and ensuring that the models specified are appropriate \cite{heath2008taxon}. 
The impact of ILS on phylogenetic inferences has been explored through simulated data sets with a known species tree. 
While the increase in the number of genes used to infer the species phylogeny reduces the interference from ILS in the results, an increase in taxa and an even distribution thereof has a greater impact \cite{maddison2006inferring}.
However the addition of taxa can also introduce more long branches that confound the inference \cite{heath2008taxon}. 
Furthermore, LBA is exacerbated if ILS is present. 
LBA can arise if some species have disproportionately high substitution rates which causes the presence of long and short branches in the phylogenetic topology \cite{liu2014coalescent}. 
The gonyaulacales data set in this study includes a single representative species per genus, with the exception of \textit{Alexandrium}, \textit{Gambierdiscus} and \textit{Ostreopsis}. 
This resulted in some genera on long branches (eg. fig. ~\ref{fig:SCmscBI} \textit{Ceratium fusus} \& \textit{Pyrodinium bahamense}) indicative of a proportionally large number of genetic changes to their closest relative.
This type of topology is indicative of sparse taxon coverage and can lead to LBA \cite{heath2008taxon}. 
To investigate the presence of ILS and as a topological comparison to the BI and ML inferences, a neighbor-joining (NJ) inference was run as well (Phylip with Protdist JTT matrix and neighbor packages \cite{felsenstein2005phylip}). 
%MF I could not find an NJ tree in the manuscript (could be added in the supp results). Was it the tree with the negative branch lengths you showed me in Aaron's office?
The rationale for including this method was that NJ can recover an accurate species topology despite ILS where ML would fail  \cite{mendes2017concatenation}.
However NJ is more susceptible to LBA than ML or BI methods. 
The resulting topology was sufficiently anomalous, with out and in-groups clustering together, that led us to exclude this result from further discussion. 
Both BI and ML are more robust to the effects of LBA than NJ, where BI tends to outperform ML especially if the latter is performed conjunction with concatenation \cite{kubatko2007inconsistency,roch2015likelihood}. 
\subsubsection*{Quality of transcriptome assemblies.}
When using publicly available data sets, quality assessment is essential. 
Since the MMETSP data sets were made available, several studies have utilized a broader range of taxa to explore evolutionary stories involving the gonyaulacales. 
However, these have relied on the assemblies supplied as part of the project. 
The stringency for quality trimming of RNA-seq libraries prior to assembly plays a role in the number of unique contigs recovered and the subsequent assembly quality of transcriptomes. 
Regarding the transcriptome assembly method, Cohen et al. evaluated the publicly available assemblies from MMETSP using BUSCO scores, compared to processing and re-assembly with Trinity \cite{cohen2018mmetsp}. 
Cohen et al. demonstrated that while the raw data available from the MMETSP project is an excellent resource, the assemblies available as part of the project are of a lower quality than what can be achieved with current methods. 
Another factor in assembly quality is RNA-seq data processing prior to assembly, especially trimming. 
Commonly high stringency is favored, however MacManes (2014) found that this can be detrimental to the assembly and the quality cut off scores used in this study were based on those recommendations \cite{macmanes2014optimal}.
In short, the trimming and assembly pipeline used for the assemblies available as part of MMETSP have become outdated and this is reflected in the quality comparison conducted by Cohen et al. 
To address this problem, trimming and assembly of RNA-seq libraries using Trimomatic and Trinity respectively were considered integral to the transcriptome assembly in this study.
\paragraph*{Assembly parameters.}
Trinity was chosen as the assembler for this study based on the findings of Honaas et al. (2016), in which Trinity was one of the top performing assemblers for \textit{de novo} transcriptomes as tested with \textit{Arabidopsis thaliana}. 
While this plant model has similar genetic features to dinoflagellates, the conclusion of adequate comparison may be flawed.
Further, Trinity performed well for identifying isoforms of genes and excelled at assembling highly expressed genes \cite{honaas2016selecting}.
Conversely, Cerveau et al. (2016) found that Trinity, CLC Bio and IDBA-Tran assemblies all contain errors introduced by the assembly algorithms. 
Using a combination of all three assemblers yielded a final assembly closer to biological reality than any individual assembler, when no reference genome is available \cite{cerveau2016combining}.
As this study uses Trinity exclusively, it is subject to the type of errors found by Cerveau et al. (2016) which could affect downstream analysis.
\subsubsection*{Selection of paralogs to infer species evolution.}
\FloatBarrier
Inclusion of genes which diverged through a process other than speciation events, such as paralogs, violates the assumptions of commonly used phylogenetic models which assume all genes analysed have an orthologous relationship.
Hence the selection of orthologs, and the elimination of paralogs, is essential.
Paralogy is particularly rife in the dinoflagellates, and the order gonyaulacales therein. 
This study sought to address the issues arising from paralog comparison by isolating single copy genes, using the curated BUSCO gene collection and software to facilitate the process. 
As BUSCO uses lineage specific profile HMM libraries designed to target single copy genes, and the output distinguishes between single copy genes and duplications, it presents a method for reliably screening for single copy genes for phylogenomics \cite{waterhouse2017busco}.
Only one other study by Price et al. (2017) sought to address the issue of paralogy within the dinoflagellates, by selecting single genes as input. 
However there are several issues with the study presented by Price et al. \cite{price2017robust}. 
The assemblies used are from the MMETSP project, which employed outdated assembly methods as mentioned previously, and hence could present single copy genes which are instead hybrids of paralogs. 
Further, the methodology for identifying and selecting the single copy genes was not described in the publication and is not available upon request. 
%MF the author did not want to tell you how the analysis was done???
Correspondence with the author of that study established that there was no documentation for the commands that the authors executed, so no record of how the genes were attained was available. 
Hence it was impossible to compare the parameters that went into identifying and screening for single copy genes and it was not possible to scrutinize or reproduce the study.
Lastly, in that study the genes were concatenated and the evolutionary relationships were inferred using ML. 
While the node bootstrap values are high, that could be due to systematic error as outlined in previous sections.
The difference in tree topology inferred by Price et al. and by our study lies in the organization of sister taxa (Fig. ~\ref{fig:tangle}). 
The Price et al. study placed \emph{Alexandrium} spp. as the closest genus to \emph{Gambierdiscus}, while this study places \emph{Pyrodinium} as sister. 
As some \emph{Gambierdiscus} spp. produce polyketide toxins which cause ciguatera fish poisoning, establishing the close relations to \emph{Gambierdiscus} is important for investigating the toxin evolution \cite{pawlowiez2014transcriptome}.
The placement of the genus \emph{Azadinium} is also a point of difference, as Price et al. place it as part of the gonyaulacales, while this study firmly places this genus as an outgroup with \emph{Dinophysis} spp. and \emph{Karenia} spp.
As \emph{Azadinium} spp. also produce polyketide toxins that cause azaspiracid shellfish poisoning, their placement is important for investigating polyketide toxin evolution \cite{meyer2015transcriptomic}.
 
\begin{figure} 
\includegraphics[scale=.23]{figures/Price-comparison.png}
%MF: is it MSC?
\caption{Tanglegram of topology presented in this study (a); and Price et al. (2017) (b). Taxa not common to either study not shown and due to the reduced topologies from the original studies, bootstrap values are not included.} 
\label{fig:tangle}
\end{figure} 
\FloatBarrier
%MF This section will be a bit empty without SS
\subsubsection*{Model selection for inference.} The issue of model choice is an important one, as the choice of model severely influences the resulting topology. 
Mis-specification of the model, or individual parameters, can lead to a well supported but erroneous result. 
While models are a simplistic approximation of the underlying biological drivers for evolutionary processes, getting as close an approximation as possible is essential \cite{box1979all}. 
However under- and over-parameterization have been shown to impact topology and PPs to varying degrees, in and outside the Felsenstein zone \cite{lemmon2004importance}. 
%Stepping stone (SS) sampling compares the marginal likelihood of both the prior and the posterior of a model, while penalizing for overparameterization and can be used to compare the fit of one model compared to another for a given data set \cite{xie2010improving}. 
%To compare how well concatenation vs. MSC fits the single copy gene data set used in this study, SS comparison was conducted.

\subsection*{Comparison to commonly employed models and data sets}
%compare to rDNA
\subsubsection*{Phylogenetic inference using ribosomal genes.}
\FloatBarrier 
Using LSU or SSU rDNA regions for phylogenetics is common practice, at times supplemented with a small number of other genes \cite{shalchian2006combined,zhang2007three,saldarriaga2004molecular,murray2005improving,hoppenrath2010dinoflagellate}. 
It is important to acknowledge that these represent the evolutionary history of highly conserved genes, which does not necessarily represent the species evolution and assumptions of their congruence is statistically inadequate \cite{degnan2009gene}.
Utilization of rDNA loci made sense when study designs were bound by sequencing and computational limitations, as they were a universally available proxy which was informative. 
Due to this legacy they continue to be commonly employed, exclusively or as part of a subset of genes for analysis, for the gonyaulacales. 
Comparing the topology from a rDNA ML inference with the single gene copy MSC phylogeny presented here (Fig. ~\ref{fig:tanglerDNA}) shows that most clades in both topologies were completely or very well supported. 
Within the genera \emph{Gambierdiscus} and \emph{Ostreopsis}, the species resolution differed between the two data sets. 
In several cases, the placement of sister taxa was incongruous between the two analyses. 
For example, the rDNA concatenation data set places \emph{Ceratium} \& \emph{Gonyaulax} as well as \emph{Alexandrium} and \emph{Gambierdiscus} as sister taxa, while the single copy gene data set under MSC places \emph{Gonyaulax} with \emph{Protoceratium} and \emph{Gambierdiscus} with \emph{Pyrodinium}. 
This is an example of how using rDNA segments as a proxy for species evolution produces different results than an analysis of single-copy protein coding genes.

\begin{sidewaysfigure} 
\includegraphics[scale=.3]{figures/MSC-BI_vs_rDNA-ML.png} 
\caption{Tanglegram showing the topological differences in phylogenies from (A) concatenated rDNA genes (SSU and D1-D3 LSU) inferred with ML; and (B) MSC inference with 58 single copy genes. Gonyaulacales (\#16) in purple, outgroups (\#3) in light blue and taxa \textit{incertae sedis} (\#1) in teal.} 
\label{fig:tanglerDNA}
\end{sidewaysfigure} 
\FloatBarrier

%compare to concat ML
\subsubsection*{Concatenating selected genes and using ML methods for species inference.}
\FloatBarrier
Concatenation of alignments coupled with ML inference is a commonly used method as it is less computationally demanding than BI methods. 
However as demonstrated by Kubatko et al. (2007) and Roch et al. (2015), this approach is error prone. 
Concatenation assumes uniform evolutionary history across genes, with a small amount of variation possible - however this still averages the evolutionary rate for all the input genes which doesn't allow for drastically divergent gene histories \cite{roch2015likelihood}. 
The combination of concatenation and ML for phylogentic inference can result in high bootstrap values for incorrectly resolved clades, over inflating confidence in erroneous topologies \cite{degnan2009gene}. 
The application of concatenation in combination with ML is common practice in phylogenetic studies for gonyaulacoids  \cite{shalchian2006combined,zhang2007three,saldarriaga2004molecular,murray2005improving,hoppenrath2010dinoflagellate}.
We investigated whether use of a technique explicitly designed to handle multiple genes to estimate species trees would yield different results than concatenation and ML. 
A comparison between a BI inference under MSC compared to concatenated ML inference on the same single copy gene data set shows differences in topology (Fig. ~\ref{fig:tangleconcatML}). 
The species resolution within the genera \emph{Alexandrium}, \emph{Gambierdiscus} and \emph{Ostreopsis} matches between the two inference methods. 
The major difference is of the \emph{Pyrodinium} placement, where the BI MSC approach places the genus sister to \emph{Gambierdiscus} while the concatenated ML approach places it with \emph{Gonyaulax} and \emph{Protoceratium}. 
Further, the deeper branches of the phylogenies differ. 
The BI MSC method clusters \emph{Ceratium}, \emph{Gonyaulax}, \emph{Protoceratium} and \emph{Thecadinium} as a clade, while the concatenated ML approach clusters \emph{Gonyaulax}, \emph{Protoceratium} and \emph{Pyrodinium} as a clade to which \emph{Thecadinium} and then \emph{Ceratium} feature as ancestral genera. 

\begin{sidewaysfigure} 
\includegraphics[scale=.3]{figures/MSC-BI_vs_singlecopy-concat-ML.png} 
\caption{Tanglegram showing the topological differences in phylogenies with same 58 single copy gene alignments as input. (A) concatenated ML inference; and (B) MSC *BEAST2 inference. Gonyaulacales (\#16) in purple, outgroups (\#3) in light blue and taxa \textit{incertae sedis} (\#1) in teal.} 
\label{fig:tangleconcatML}
\end{sidewaysfigure} 
\FloatBarrier

%compare to concat BI

\subsubsection*{Concatenating selected genes and using BI methods for species inference.}
\FloatBarrier
Even within a BI framework concatenation can introduce a number of errors. 
Under simulated data sets, even under the coalescent methods, the species tree topology is inaccurate when concatenation is used \cite{kubatko2007inconsistency}. 
Further to that, the PP values tend to be overestimated for concatenation \cite{suzuki2002overcredibility}. 
Theoretically for the gonyaulacales, and taxa prone to paralogy and convoluted evolutionary histories, the MSC is a preferable approach to concatenation as MSC is more robust to ILS \& LBA artifacts \cite{liu2014coalescent}. 
To isolate the effects of phylogenetic model from those of the statistical framework (ML vs BI), the single copy gene data set was run with BI both under MSC and with concatenation (Fig. ~\ref{fig:tangleconcatBI}). 
%We then used a statistical framework to compare the two model approaches to verify the veracity of model adequacy through stepping stone sampling. 
%PATH sampling is an adequate method for objectively comparing the model parameters for relaxed clock models while penalizing for over-parameterization \cite{baele2012accurate}. %TODO same for ss?
%The value XXX %TODO whatever the hell the PATH sampling results are
%TODO topology comparison here 
The resolution of \textit{Alexandrium}, \textit{Coolia} and \textit{Ostreopsis} was identical between the two methods. 
Further, the species resolution within the genera \textit{Gambierdiscus} and \textit{Ostreopsis} was also identical. 
Differences were found in the topology, in that \textit{Pyrodinium} clustered with \textit{Gambierdiscus} in the MSC analysis, while for concatenation this genus clusters with \textit{Gonyalax} and \textit{Protoceratium}. 
% AD 181018: is the pyrodinium placement the same as ML? Same as Price et al?
Further, in the MSC analysis \textit{Ceratium}, \textit{Gonyalax}, \textit{Protoceratium} and \textit{Thecadinium} form their own clade while with concatenation, \textit{Ceratium} and \textit{Thecadinium} are ancestral genera to the rest of the gonyaulacales. 

%TODO CHeck again after ss sampling is complete and also if run0 is posterior
\begin{sidewaysfigure} 
\includegraphics[scale=.25]{figures/SC-MSC-BI_vs_SC-concat-BI.png} 
\caption{Tanglegram showing the topological differences in phylogenies with same 58 single copy gene alignments as input. (A) concatenated BEAST2, rerooted on the branch separating outgroups from the onyaulacales; and (B) MSC *BEAST2 inference. Gonyaulacales (\#16) in purple, outgroups (\#3) in light blue and taxa \textit{incertae sedis} (\#1) in teal.} 
\label{fig:tangleconcatBI}
\end{sidewaysfigure} 
\FloatBarrier

\subsection*{Areas for possible improvement of this study}
In the previous section we identified potential problems with common approaches to species inference in the literature, and in particular for the gonyaulacales. 
We then sought to evaluate the effects of different methodological approaches on analytical results in the gonyaulacales. 
There are several important limitations to our study. 
\paragraph*{Contamination of other taxa.} 
The 650+ RNA extract submission to MMETSP was from a large number of investigators and low level contamination is inherent in the project's data set \cite{keeling2014marine}. 
As the cultures tested in all the studies contributing to this data set were not axenic, contamination could be bacterial or eukaryotic in nature. 
While any contaminating bacterial genes in our data would likely be heavily diverged and therefore obvious, eukaryotic contamination may be more subtle.
\paragraph*{No representative genome for comparison.} 
Without an available reference genome, it is difficult to evaluate the accuracy of the transcriptome assembly and whether the genes selected are single copies, or misassemblies of paralogs.
\paragraph*{Different methods for RNA-seq.} 
Three different approaches for RNA-seq library generation were employed for the libraries used in this study, the MMETSP taxa were sequences on HiSeq platform with 50nt reads; while all other taxa were sequenced on the NextSeq platform with 75nt or 150nt reads. 
The different sequencing methods may each influence the single copy gene coverage and transcriptome assembly accuracy, leading to systematic error and batch effects on some taxa.
\paragraph*{Total evidence phylogenetics.}
The method presented here purely considers the information contained in the genetic aspect of the organisms examined. 
Morphological characters and fossil dates can add another dimension to the phylogenetic inference and put the evolution within a relative time frame \cite{gavryushkina2017bayesian}.  
\paragraph*{Model comparison methods.}
Recent advances in computational statistics have yielded methods such as path sampling and the stepping stone, which facilitate model comparison via marginal likelihood estimation. 
Application of these methods could in principle provide an objective means to determine whether the MSC model is better supported than concatenation, for example.
However, marginal likelihood estimation is extremely demanding, requiring months of computation even with GPU accelleration.
Therefore comparison of model marginal likelihoods remains as future work.

\newpage
\section{Conclusion}
With the public accessibility of large data sets, the focus of the scientific community has begun to shift toward the methodology required to analyze them. 
This study presents a workflow for species tree inference that implements what is currently thought to be the best practice methods. 
The scripts process RNA-seq libraries through assembly, single copy gene selection to alignment for phylogenetic species inference. 
As a case study exemplifying organisms rife with paralogs and ancient lineages, the gonyaulacales were selected. 
The resulting phylogeny shows a well resolved, well supported inference of the gonyaulacales evolution. 
This was then compared to phylogenies inferred from commonly utilized methods in the literature, and potential issues arising from these methods were discussed. 
By presenting a statistically rigorous method and demonstrating how it overcomes common problems in phylogenetic studies, we hope that in the future such robust, reproducible, open-access approaches to process large data-sets such as the MMETSP database can become standard practice.
\newpage

\section{Acknowledgments}
The GVL section of this study was conducted inside the National eResearch Collaboration Tools and Resources (NeCTAR) research cloud, an initiative by the National Research Infrastructure for Australia (NCRIS).
Gratitude to the Stanley Watson foundation, the Linnaean Society of New South Wales, and the ABRS National Taxonomy Research Student Travel Bursary for funding A. L. Kretzschmar's attendance at the Molecular Evolution workshop at the Marine biological laboratory, Woods Hole, MA, USA.
Shout out to the Taming the BEAST organizers \& fellow attendees for a most illuminating workshop in February 2017 on BEAST methodology, and to Geneious for subsidizing A. L. Kretzschmar's attendance fee.
Thank you to Dr. Tim Kahlke for running Interproscan for transcriptome analysis. 
The transcriptomic sequencing was funded by an ARC Future Fellowship to S. Murray.
A. L. Kretzschmar's PhD stipend was funded through a UTS Doctoral scholarship.
\section{Supplementary material}
\FloatBarrier
\begin{table}
\caption{Table S1: Culturing conditions for species processed for this study.}
%\label{tbl:strainTable}
\begin{tabular}{ | p{3cm} | p{2.5cm} | p{1.5cm} | p{5.3cm} |}
\hline
\textbf{Species} & \textbf{Strain}& \textbf{Temp} & \textbf{Source location} \\
%\hline
%\textit{Coolia malayensis}&MAB&& \\
\hline
\textit{Gambierdiscus carpenteri}&UTSMER9A&17&Merimbula, AU\\
\hline
\textit{Gambierdiscus lapillus}&HG4&27&Heron Island, AU\\
\hline
\textit{Gambierdiscus polynesiensis}&CG15&27&Rarotonga, COK\\
\hline
\textit{Gambierdiscus} cf. \textit{silvae}&HG5&27&Heron Island, AU\\
\hline
%\textit{Ostreopsis ovata}&HER27&&\\
%\hline
%\textit{Ostreopsis rhodesae}&HER26&&\\
%\hline
%\textit{Ostreopsis siamensis}&BH1&&\\
%\hline
\textit{Thecadinium} cf. \emph{kofoidii}&THECA&18&Gordons bay, Sydney, AU\\
\hline
\end{tabular}
\end{table}
\FloatBarrier

\begin{longtable}{  | p{3.5cm} |p{2.2cm} | p{1.8cm} | p{1.8cm} | p{1.8cm} | p{3cm} |}
\caption{Table S2: Transcriptomes used for study along including strain ID, source and BUSCOv2 information. MMETSP abbreviation for marine Microbial eukaryotic transcriptome sequencing project, by Moore Foundation.}\\
\hline
%\label{tbl:Transcriptomes}
%\textbf{Family}&
\textbf{Species}&\textbf{Strain}&\textbf{complete BUSCOs}&\textbf{single complete BUSCOs}&\textbf{fragmented BUSCOs}&\textbf{Source}\\
\hline
 \multicolumn{6}{| c |}{Gonyaulacales transcriptomes}\\
    \hline
\emph{Alexandrium catenella}&OF101&110&74&3&MMETSP0790 \citep{keeling2014marine}\\
        \hline
\emph{Alexandrium monilatum}&JR08&107&74&3&MMETSP0093 \citep{keeling2014marine}\\
        \hline
\emph{Ceratium fusus}&PA161109&121&81&4&MMETSP1074 \citep{keeling2014marine}\\
        \hline
\emph{Coolia malayensis}&MAB&138&100&1&(Verma 2018, in prep)\\
\hline
\emph{Crypthecodinium cohnii}&Seligo&126&98&0&MMETSP0326\_2 \citep{keeling2014marine}\\
        \hline
\emph{Gambierdiscus carpenteri}&UTSMER9A&101&83&2&This study\\
\hline
\emph{Gambierdiscus excentricus}&VGO790&88&83&4&\cite{kohli2017role}\\
        \hline
\emph{Gambierdiscus lapillus}&HG4&141&98&2&This study\\
        \hline
\emph{Gambierdiscus polynesiensis}&CG15&104&81&3&This study\\
        \hline
\emph{Gambierdiscus} cf. \emph{silvae}&HG5&134&87&2&This study\\
        \hline
\emph{Gonyaulax spinifera}&CCMP409&83&53&2&MMETSP1439 \citep{keeling2014marine}\\
        \hline
\emph{Ostreopsis ovata}&HER27&132&99&2&(Verma 2018, in prep)\\
     \hline
\emph{Ostreopsis rhodesae}&HER26&131&98&1&(Verma 2018, in prep)\\
     \hline
\emph{Ostreopsis siamensis}&BH1&132&98&1&(Verma 2018, in prep)\\
     \hline
\emph{Protoceratium reticulatum}&CCCM535=CCMP1889&108&72&5&MMETSP0228 \citep{keeling2014marine}\\
    \hline
\emph{Pyrodinium bahamense}&pbaha01&119&897&2&MMETSP0796 \citep{keeling2014marine}\\
        \hline
\emph{Thecadinium} cf. \emph{kofoidii}&THECA&93&70&5&This study\\
 \hline
 \multicolumn{6}{| c |}{Outgroup transcriptomes}\\
 \hline
 \emph{Azadinium spinosum}&3D9&1.8&81&4&MMETSP1036\_2 \citep{keeling2014marine}\\
        \hline
\emph{Dinophysis acimunata}&DAEP01&117&74&2&MMETSP0797 \citep{keeling2014marine}\\
        \hline
\emph{Karenia brevis}&CCMP2229&115&85&2&MMETSP0030 \citep{keeling2014marine}\\
    \hline
\end{longtable}
\FloatBarrier
%families, sections which was taken out:Ceratiaceae,Crypthecodiniaceae,Gonyaulacaceae,Protoceratiaceae,Dinophysiaceae,Dinophyceae incertae sedis,Gymnodiniales

\begin{longtable}{  | p{3cm} |p{3cm} |  p{3cm} | }
\caption{Table S3: Accession numbers for ribosomal DNA sequences used for Fig. ~\ref{fig:rdna}. Sequences sourced from NCBI, except accesion numbers with '$\ast$' sourced from the Silva database. Genes not publically available are denoted by '-'.}\\
\hline
%\label{tbl:Transcriptomes}
%\textbf{Family}&
\textbf{Species}&\textbf{SSU seq.}&\textbf{D1-D3 LSU seq.}\\

\hline
 \multicolumn{3}{| c |}{Gonyaulacales taxa}\\
 \hline
\emph{Alexandrium catenella}&AB088286&AB088238\\
        \hline
\emph{Alexandrium monilatum}&AY883005&-\\
        \hline
\emph{Ceratium fusus}&AF022153&AF260390\\
        \hline
\emph{Coolia malayensis}&HQ897279$\ast$&KX589143\\
\hline
\emph{Crypthecodinium cohnii}&M64245&-\\
        \hline
\emph{Gambierdiscus carpenteri}&EF202908&EF202938\\
\hline
\emph{Gambierdiscus excentricus}&GETL01000157$\ast$&HQ877874\\
        \hline
\emph{Gambierdiscus lapillus}&KU558930&-\\
        \hline
\emph{Gambierdiscus polynesiensis}&EF202907&This study\\
        \hline
\emph{Gambierdiscus} cf. \emph{silvae}&This study&this study\\
        \hline
\emph{Gonyaulax spinifera}&AF022155&DQ151558\\
        \hline
\emph{Ostreopsis ovata}&AF244939&KJ781420\\
     \hline
\emph{Ostreopsis rhodesae}&KX055855&KX055845\\
     \hline
\emph{Ostreopsis siamensis}&KX055868&HQ414223\\
     \hline
\emph{Protoceratium reticulatum}&AF274273&EF613362\\
    \hline
\emph{Pyrodinium bahamense}&AY456115&AB936757\\
        \hline
\emph{Thecadinium} cf. \emph{kofoidii}&AY238478&KT371445\\
 \hline
\multicolumn{3}{| c |}{Outgroup taxa}\\
    \hline
  \emph{Azadinium spinosum}&JN680857&JN165101\\
        \hline
\emph{Dinophysis acimunata}&AJ506972&EF613351\\
        \hline
\emph{Karenia brevis}&EF492504&AY355458\\
\hline
\end{longtable}
\FloatBarrier

\newpage
\bibliographystyle{acm}
\bibliography{gonya.bib}

\end{document}